pkg/reconciler/clusterimagepolicy/clusterimagepolicy.go

```diff
@@ -28,13 +38,67 @@ import (
 type Reconciler struct {
 	// Tracker builds an index of what resources are watching other resources
 	// so that we can immediately react to changes tracked resources.
-	Tracker tracker.Interface
+	tracker tracker.Interface
+	// We need to be able to read Secrets, which are really holding public
+	// keys.
+	secretlister    corev1listers.SecretLister
+	configmaplister corev1listers.ConfigMapLister
+	kubeclient      kubernetes.Interface
 }
 
 // Check that our Reconciler implements Interface
 var _ clusterimagepolicyreconciler.Interface = (*Reconciler)(nil)
 
 // ReconcileKind implements Interface.ReconcileKind.
-func (r *Reconciler) ReconcileKind(ctx context.Context, o *v1alpha1.ClusterImagePolicy) reconciler.Event {
+func (r *Reconciler) ReconcileKind(ctx context.Context, cip *v1alpha1.ClusterImagePolicy) reconciler.Event {
+	if !willItBlend(cip) {
+		return errors.New("i can't do that yet, only support keys inlined or KMS")
+	}
+
+	// See if the CM holding configs exists
+	existing, err := r.configmaplister.ConfigMaps(SystemNamespace).Get(config.ImagePoliciesConfigName)
+	if err != nil {
+		if !apierrs.IsNotFound(err) {
+			logging.FromContext(ctx).Errorf("Failed to get configmap: %v", err)
+			return err
+		}
+		// Does not exist, create it.
+		cm, err := resources.NewConfigMap(SystemNamespace, config.ImagePoliciesConfigName, cip)
+		if err != nil {
+			logging.FromContext(ctx).Errorf("Failed to construct configmap: %v", err)
+			return err
+		}
+		_, err = r.kubeclient.CoreV1().ConfigMaps(SystemNamespace).Create(ctx, cm, metav1.CreateOptions{})
```

Comment:
I'm wondering if we should assume it always exists since it is part of `config`? 🤔

Reply:
The one thought is that if somebody somehow deletes it by accident? I can remove it though, just seemed safer. Just lmk if you want me to remove.

Reply:
I don't feel super strongly either way.  It would let us drop `create` from the controller's RBAC, but otherwise it's pretty benign